src/index.js

```diff
@@ -66,9 +68,9 @@ class App {
 
 const scenes = [
   // First scene gets loaded first
-  LobbyScene,
```

Comment:
Are you wanting todo this all in one scene? Seems like we would want a scene for the lobby/menu/game

Reply:
No, we will have multiple scenes, I just commented the others out for testing